src/components/Toast/use-toast.tsx

```diff
@@ -0,0 +1,188 @@
+// Inspired by react-hot-toast library
+import * as React from "react";
+
+import type { ToastActionElement, ToastProps } from "./Toast";
+
+const TOAST_LIMIT = 2;
+const TOAST_REMOVE_DELAY = 1000000;
```

Comment:
16 min and 40 secs, is this not too much? 😅

Reply:
you're actually correct, this is used inside a `setTimeout` it seems to be the default though. And where its used it seems fine 🤔